proto/storage/compliance_operator_v2.proto

```diff
@@ -97,6 +97,10 @@ message ComplianceOperatorScanConfigurationV2 {
     // Most recent user to update the scan configurations
     SlimUser                      modified_by              = 14 [(gogoproto.moretags) = 'sql:"ignore_labels(User ID)"'];
     string                        description              = 15;
+    message Cluster {
```

Comment:
for my own understanding, why not just do a repeated string?

Reply:
I tried that and similar but I don't recall what the exact issue was.  I will experiment with that again, but if memory serves the SAC filter does not play well with that.

Reply:
@theencee Now I remember why that doesn't work.  The generated store does not handle that type in regards to the sac filter.